Formula/rust.rb

```diff
@@ -40,8 +40,8 @@ class Rust < Formula
 
   resource "cargobootstrap" do
     # From https://github.com/rust-lang/rust/blob/#{version}/src/stage0.txt
-    url "https://static.rust-lang.org/dist/2019-12-19/cargo-0.41.0-x86_64-apple-darwin.tar.gz"
-    sha256 "1ef77a6be5e697bb7dde40854651fc67e91f119d5a9ddf747a25e30c1179fbe1"
+    url "https://static.rust-lang.org/dist/2020-03-12/cargo-0.43.0-x86_64-apple-darwin.tar.gz"
+    sha256 "92d4c9fb4747dce158cdfb773651aea8eac894277f3a2de5aa2c3b9d92439d8e"
```

Comment:
No, this one should be 0.42.0 as per the comment.

https://github.com/rust-lang/rust/blob/1.42.0/src/stage0.txt

Reply:
Somehow I could not find the link for `cargo-0.42.0-x86_64-apple-darwin.tar.gz`, feel free to update it.

Reply:
https://static.rust-lang.org/dist/2020-02-27/cargo-0.42.0-x86_64-apple-darwin.tar.gz

Reply:
You are awesome!

Reply:
Just curious, how did you find it?

I tried the date listed on the cargo github releases page, it did not quite work.

Reply:
I used the date in the stage0.txt file.

Reply:
I see